ereol_app_feeds/src/Feed/FrontPageFeed.php

```diff
@@ -90,16 +90,13 @@ private function getThemes(array $paragraphIds) {
   /**
    * Get reviews.
    *
-   * @param array $paragraphIds
-   *   The paragraph ids.
-   *
    * @see https://docs.google.com/document/d/1lJ3VPAJf7DAbBWAQclRHfcltzZefUG3iGCec-z97KlA/edit?ts=5c4ef9d5#bookmark=id.qh5qjlx68dde
    *
    * @return array
    *   The reviews data.
    */
-  private function getReviews(array $paragraphIds) {
-    return $this->paragraphHelper->getParagraphsData(ParagraphHelper::PARAGRAPH_REVIEW, $paragraphIds);
+  private function getReviews() {
```

Comment:
Would it not be more flexible if it had  `private function getReviews(int $no_of_reviews = 5)` or would that be overkill.

Reply:
@cableman, it definitely would be more flexible, but for now we don't need this flexibility. And probably never will.

Reply:
👍